### ISSUE TITLE
Warnings while building step our website for outdated links within Markdown links

### ISSUE THREAD

**Issue by FaGru3n** [eclipse-tractusx/eclipse-tractusx.github.io#639]
### Current Behavior

While you build local the website there apprear a lot warnings for Markdown Links within our documentation and some files

### Expected Behavior

Less or no warnings while building step

### Steps To Reproduce

- check out repo
- run `npm run build`
- validate the warnings

### Examples 
```
[WARNING] Docs markdown link couldn't be resolved: (../page_adoption-view.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Traceability Kit/Software Development View/part_digital-twin.mdx" for version current
[WARNING] Docs markdown link couldn't be resolved: (../edc-extensions/hashicorp-vault/README.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/tractusx-edc/docs/README.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (/docs/kit-process/contribute.md) in "eclipse-tractusx.github.io/docs/kit-process/processes/kit-faq.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../development-view/openAPI/management-api/management-api.info.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.09/kits/tractusx-edc/docs/kit/operation-view/page08_api.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../development-view/openAPI/management-api/management-api.info.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/docs/kit/operation-view/page08_api.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../../edc-tests/cucumber/src/main/resources/deployment/helm/supporting-infrastructure/README.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/docs/samples/Local TXDC Setup.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../docs/development/Run-business-tests-local.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/edc-tests/cucumber/README.md" for version current
[WARNING] Docs markdown link couldn't be resolved: (../Software%20Development%20View/Gate%20Api/put-addresses-output.api.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Business Partner Kit/Software Development View/page_software-development-view.md" for version current
[WARNING] The image at "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_SequenceChart.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_SequenceChart.svg)
[WARNING] The image at "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_Overview.svg" can't be read correctly. Please ensure it's a valid image.
unsupported file type: undefined (file: eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_Overview.svg)
```

**Comment by FaGru3n** [comment]
Hi @danielmiehle @maximilianong just found some warnings while building step within our documentation
could you overtake this or recheck the documentation and the markdown links within our Markdown Files

[warnings.md](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/files/14094663/warnings.md)

**Comment by maximilianong** [comment]
Hi @FaGru3n we need to separate those warnings and assign it to each responsible. I see warnings in the modular production KIT, in the traceability KIT .. they should be solved by the KIT responsibles.

Were they always there, or is there a reason they're coming up now?

**Comment by FaGru3n** [comment]
hi @maximilianong thanks for feedback, guess there were allways there an now will sum up with more and more releases.

So baiscally want to share that there are some outdated links within our Markdown files and at least tried to help with the PR.

After some investigations found some missleading information and thats why i raised the issue to inform you and for myself i can not decide with links will be present or need an update and link to the correct version for the docs.

**Comment by maximilianong** [comment]
Daniel and me separated the warnining and assigned it to the resposibles - please take care of the warnings and fix it.


- [ ] [WARNING] Docs markdown link couldn't be resolved: (../page_adoption-view.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Traceability Kit/Software Development View/part_digital-twin.mdx" for version current
@mhellmeier 


- [ ] [WARNING] Docs markdown link couldn't be resolved: (../edc-extensions/hashicorp-vault/README.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/tractusx-edc/docs/README.md" for version current
@stephanbcbauer / @stefan-ettl 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (/docs/kit-process/contribute.md) in "eclipse-tractusx.github.io/docs/kit-process/processes/kit-faq.md" for version current
This one will be deleted by @maximilianong / @danielmiehle 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (../development-view/openAPI/management-api/management-api.info.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.09/kits/tractusx-edc/docs/kit/operation-view/page08_api.md" for version current
@stephanbcbauer / @stefan-ettl 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (../development-view/openAPI/management-api/management-api.info.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/docs/kit/operation-view/page08_api.md" for version current
@stephanbcbauer / @stefan-ettl 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (../../edc-tests/cucumber/src/main/resources/deployment/helm/supporting-infrastructure/README.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/docs/samples/Local TXDC Setup.md" for version current
@stephanbcbauer / @stefan-ettl 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (../docs/development/Run-business-tests-local.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-3.1.0/kits/tractusx-edc/edc-tests/cucumber/README.md" for version current
@stephanbcbauer / @stefan-ettl 

- [ ] [WARNING] Docs markdown link couldn't be resolved: (../Software%20Development%20View/Gate%20Api/put-addresses-output.api.mdx) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Business Partner Kit/Software Development View/page_software-development-view.md" for version current
@SujitMBRDI 

- [ ] [WARNING] The image at "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_SequenceChart.svg" can't be read correctly. Please ensure it's a valid image.
@dominikoeh 

- [ ] unsupported file type: undefined (file: eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_SequenceChart.svg)
@dominikoeh 

- [ ] [WARNING] The image at "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_Overview.svg" can't be read correctly. Please ensure it's a valid image.
@dominikoeh 

- [ ] unsupported file type: undefined (file: eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Modular Production Kit/res/SIS_ProduktionForecast_Overview.svg)
@dominikoeh

**Comment by mhellmeier** [comment]
I will check it.

@maximilianong : Remember that there are also some broken links on https://eclipse-tractusx.github.io/developer, like the one to the "Traceability Kit" and to the "Business Domain Resiliency".

**Comment by maximilianong** [comment]
> I will check it.
> 
> @maximilianong : Remember that there are also some broken links on https://eclipse-tractusx.github.io/developer, like the one to the "Traceability Kit" and to the "Business Domain Resiliency".

Thanks.

& you are right regarding the /developer page. We will change this whole page during the next week.

**Comment by dominikoeh** [comment]
Have checked the warnings: seem to originate from svg images created with draw.io and having their diagrams included during export. 

Diagram content has been removed from exported images and has been added for later use in the resources

**Comment by stephanbcbauer** [comment]
@stefan-ettl I have fixed the errors for the connector kit

**Comment by arnoweiss** [comment]
> Daniel and me separated the warnining and assigned it to the resposibles - please take care of the warnings and fix it.
> 
> * [ ]  [WARNING] Docs markdown link couldn't be resolved: (../page_adoption-view.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Traceability Kit/Software Development View/part_digital-twin.mdx" for version current
>   @mhellmeier

This gets fixed with https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/654

**Comment by mhellmeier** [comment]
> * [x]  [WARNING] Docs markdown link couldn't be resolved: (../page_adoption-view.md) in "eclipse-tractusx.github.io/docs-kits_versioned_docs/version-23.12/kits/Traceability Kit/Software Development View/part_digital-twin.mdx" for version current
>   @mhellmeier

Fixed in #675

**Comment by FaGru3n** [comment]
@mhellmeier thx for your effort will close this issue for now.
